packages/integration-tests/api/distribution.js

```diff
@@ -108,7 +108,8 @@ function getDistributionFileUrl({
   bucket,
   key
 }) {
-  return `${distributionEndpoint}/${bucket}/${key}`;
+  const theUrl = new URL(`${bucket}/${key}`, distributionEndpoint);
```

Comment:
What is the difference between this and the previous approach?

Reply:
The previous approach puts double slashes between the endpoint and the path if the endpoint is defined with a trailing slash as we tell people to do in the configuration.  This was just the easiest way to remove a double slash between the endpoint and the bucket/key path because the endpoint also has `http://host` which I don't want to replace.